app/src/main/java/com/antosik/benchproject/app/main/view/MainActivity.kt

```diff
@@ -26,5 +28,21 @@ class MainActivity : AppCompatActivity() {
         val navController = navHostFragment.navController
         val bottomNavigationView = binding.activityMainBottomNavigation
         setupWithNavController(bottomNavigationView, navController)
+
+        navController.addOnDestinationChangedListener { _, destination, _ ->
+            when (destination.id) {
+                R.id.popularMoviesFragment -> bottomNavigationView.showBottomNavigation()
+                R.id.favouriteMovies -> bottomNavigationView.showBottomNavigation()
+                else -> bottomNavigationView.hideBottomNavigation()
+            }
+        }
+    }
+
+    private fun BottomNavigationView.showBottomNavigation() {
```

Comment:
It would be nice to create your own BottomNavigationView component and place this there. Later on, it will be a place to add an animation.

Reply:
Hmm... ok, we can talk about this 😄